### PR TITLE
Update getTokenURI()

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: certmanager/util.go [hunk 1]
```diff
@@ -33,12 +33,16 @@ type GcpTokenFetcher struct {
 	// aud is the unique URI agreed upon by both the instance and the system verifying the instance's identity.
 	// For more info: https://cloud.google.com/compute/docs/instances/verifying-instance-identity
 	aud            string
-	serviceAccount string
 }
 
 func (fetcher *GcpTokenFetcher) getTokenURI() string {
 	// The GCE metadata service URI to get identity token.
-	return "instance/service-accounts/" + fetcher.serviceAccount + "/identity?audience=" + fetcher.aud
```
Comment (wattli) [inline certmanager/util.go]: just tested it one more time, instance/service-accounts/default/identity is good enough, which gives you the unique ID corresponding to the service account running on the current VM.

File: certmanager/util.go [hunk 2]
```diff
@@ -33,12 +33,16 @@ type GcpTokenFetcher struct {
...
+	return "instance/service-accounts/default/identity?audience=" + fetcher.aud
+}
+
+func (fetcher *GcpTokenFetcher) getServiceAccount() (string, error) {
```
Comment (lookuptable) [inline certmanager/util.go]: This is a private method but is not called by anyone. Consider adding it when it's actually needed?

File: certmanager/util.go [hunk 3]
```diff
@@ -33,12 +33,16 @@ type GcpTokenFetcher struct {
 	// aud is the unique URI agreed upon by both the instance and the system verifying the instance's identity.
 	// For more info: https://cloud.google.com/compute/docs/instances/verifying-instance-identity
 	aud            string
-	serviceAccount string
 }
 
 func (fetcher *GcpTokenFetcher) getTokenURI() string {
 	// The GCE metadata service URI to get identity token.
```
Comment (lookuptable) [inline certmanager/util.go]: Can you update the comment to see the URI is for the **default** service account?

Comment (wattli) [inline certmanager/util.go]: Done.

File: certmanager/util.go (hunk 2 above)
Comment (wattli) [inline certmanager/util.go]: okay.

File: certmanager/util.go (hunk 3 above)
Comment (wattli) [inline certmanager/util.go]: This func is deleted.

**Comment by wattli** [comment]
@lookuptable , can you take another look? Just fixed the gofmt error.

File: certmanager/util.go (hunk 3 above)
Comment (wattli) [inline certmanager/util.go]: And this is a one line function, do you have any suggestion about the unit test?

**Pull request merged**